projects/media-viewer/src/lib/store/reducers/annotatons.reducer.ts

```diff
@@ -0,0 +1,208 @@
+import * as fromAnnotations from '../actions/annotations.action';
+import {Annotation} from '../../annotations/annotation-set/annotation-view/annotation.model';
+import {StoreUtils} from '../store-utils';
+import {SelectionAnnotation} from '../../annotations/models/event-select.model';
+
+export interface AnnotationSetState {
+  annotationSet: any;
+  annotationEntities: {[id: string]: any};
+  annotationPageEntities: {[id: string]: Annotation[]};
+  commentEntities: {[id: string]: Comment} | {};
+  selectedAnnotation: SelectionAnnotation;
+  pages: { numberOfPages: number; styles: any; scaleRotation: object };
```

Comment:
why not split scaleRotation into scale and rotation

Reply:
a yeah that is actually split but type is just wrong fixed now